packages/dev/core/src/Misc/basis.ts

```diff
@@ -327,22 +327,20 @@ function workerFunc(): void {
         if (event.data.action === "init") {
             // Load the transcoder if it hasn't been yet
             if (!transcoderModulePromise) {
-                // Override wasm binary
-                Module = { wasmBinary: event.data.wasmBinary };
                 // make sure we loaded the script correctly
                 try {
                     importScripts(event.data.url);
                 } catch (e) {
                     postMessage({ action: "error", error: e });
                 }
-                transcoderModulePromise = new Promise<void>((res) => {
-                    Module.onRuntimeInitialized = () => {
-                        Module.initializeBasis();
-                        res();
-                    };
+                transcoderModulePromise = BASIS({

```

Comment:
I'm aware that typescript would like to know where `BASIS` comes from, but I'm not sure how to do that properly. It gets defined in the imported script.

Reply:
You can replace `Module` with `BASIS` above. That's how you tell typescript this global object exists.

Reply:
So the latest changes should be correct, right?

Reply:
Yes, I believe so.

Reply:
Yes, I believe so.